client/src/components/User/getUserData/index.js

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from "react";
+
+export default function useFindUser() {
+  const [user, setUser] = useState(null);
+  const [isLoading, setLoading] = useState(true);
+  useEffect(() => {
+    async function findUser() {
+      // set up fetch requests
+      const requestOptions = {
+        method: "GET",
+        headers: {
+          Accept: "application/json",
+        },
+      };
+
+      await fetch("/auth", requestOptions)
+        .then(async function (res) {
+          const data = await res.json();
+          // console.log(data.user);
+          setUser(data.user);
+          setLoading(false);
+        })
+        .catch((err) => {
+          setLoading(false);
+        });
+    }
+    findUser();
```

Comment:
Is this feature in scope for this issue? Right now there's no way to test it as the backend hasn't been set up to handle that route

Reply:
It might be a bit of a stretch, but it was specified in the ticket:

`To get user information, you will need to make an API call every time you open the app, to correctly get up to date logged in user information`

I figured I should set up the api call, expect 404 for now, and ensure it integrates properly later.